packages/compile-solidity/test/test_supplier.js

```diff
@@ -11,23 +11,23 @@ function waitSecond() {
   return new Promise(resolve => setTimeout(() => resolve(), 1250));
 }
 
-describe("CompilerSupplier", function() {
-  describe("integration", function() {
+describe("CompilerSupplier", function () {
+  describe("integration", function () {
     this.timeout(40000);
     let oldPragmaPinSource; //  0.4.15
     let oldPragmaFloatSource; // ^0.4.15
     let version4PragmaSource; // ^0.4.21
     let version5PragmaSource; // ^0.5.0
-    let version6PragmaSource; // ^0.6.0
+    let version7PragmaSource; // ^0.7.0
```

Comment:
I think we should keep `version6PragmaSource` around for completeness

Reply:
Husky won't allow it; unused variable.  Should I disable the check there or something?

Reply:
We already run tests for version 4 and version 5, right? Those aren't a problem with unused variables.

What I'm proposing is that we keep the test there for 0.6 and merely _add_ tests for 0.7

Reply:
But, as I explained above, it's **not** a test of 0.6.x.  It's a test of docker/native.  It will always be the latest version.  If it were a test of 0.6.x, obviously I would have kept it and added a parallel test of 0.7.x.  But it isn't!  If it were, it wouldn't be failing in the first place!

Reply:
Oh then let's just change the variable and directory name and such

Reply:
Hm, I suppose that works.  I'll continue to keep around the 0.6.x source for now though unless you think it should be removed.

Reply:
Also I guess I'll keep around 0.7.x separate from latest.  Could come in handy.

Reply:
Hm the test explicitly checks that latest is 0.n.x though (for specified n).  Will have to figure out how to handle that.

Reply:
My point is purely to avoid future head-scratching when someone sees this code and goes "why is there no version6Pragma".

This is a minor point, though, so heed this concern only if you think you see a reasonable path forward here.

Reply:
Oh, OK.  Sure, I'll just change the variable then; I'll leave the path alone.